cmd/goal/application.go

```diff
@@ -95,6 +96,7 @@ func init() {
 	appCmd.AddCommand(methodAppCmd)
 
 	appCmd.PersistentFlags().StringVarP(&walletName, "wallet", "w", "", "Set the wallet to be used for the selected operation")
+	appCmd.PersistentFlags().Uint64Var(&rejectVersion, "reject-version", 0, "Reject for this app version or higher")
```

Comment:
Do we want to add a comment about leaving or setting to 0 is the same as not using?

Reply:
Agreed, changing to "If set non-zero, reject for this app version or higher"